dynamic_rest/viewsets.py

```diff
@@ -336,5 +336,21 @@ def list_related(self, request, pk=None, field_name=None):
         return Response(serializer.data)
 
 
+class BulkCreationMixin(object):
+    """
+    Either create a single or many model instances in bulk by using the
+    Serializer's many=True ability from Django REST >= 2.2.5.
+    """
+    def create(self, request, *args, **kwargs):
+        if not isinstance(request.data, list):
+            return super(BulkCreationMixin, self).create(
+                request, *args, **kwargs)
+        else:
+            serializer = self.get_serializer(data=request.data, many=True)
+            serializer.is_valid(raise_exception=True)
+            self.perform_create(serializer)
+            return Response(serializer.data, status=status.HTTP_201_CREATED)
```

Comment:
What happens if there a partial failure, e.g. one out of 3 posted object failed?